astropy/nddata/utils.py

```diff
@@ -20,16 +53,26 @@ def overlap_slices(large_array_shape, small_array_shape, position):
     used to extract, add or subtract the small array at the given
     position. This function takes care of the correct behavior at the
     boundaries, where the small array is cut of appropriately.
+    Integer positions are at the pixel centers.
 
     Parameters
     ----------
-    large_array_shape : tuple
-        Shape of the large array.
-    small_array_shape : tuple
-        Shape of the small array.
-    position : tuple
+    large_array_shape : tuple or number
+        Shape of the large array (for 1D arrays, this can be an int or float).
+    small_array_shape : tuple or number
+        Shape of the small array (for 1D arrays, this can be an int or float).
+    position : tuple of integers or int
```

Comment:
This isn't actually restricted to  integers, right? How about just `tuple or number` here, along with the minor edit a few lines down?